source/write_docs/content/index.html.md.erb

```diff
@@ -108,36 +65,64 @@ The following code is an example of a 2 column 2 row table with no text wrapping
 </div>
 ```
 
-## Links
+## Create a link
 
-```bash
-[LINK TEXT](LINK ADDRESS)
+You can create links using either markdown or HTML.
+
+To create a link in markdown, use the following:
+
+```yaml
+[link text](link address)
 ```
 
 For example:
 
-```bash
+```yaml
 [The Google home page](http://www.google.co.uk)
 ```
 
+To create a link in HTML, use the following:
+
+```bash
+<a href="link address">link text</a>
+```
+
 You can write the link in HTML with a `target="blank"` element to make the link open in a new internet browser tab by default:
 
 ```bash
-<a href="LINK ADDRESS" target="blank">LINK TEXT</a>
+<a href="link address" target="blank">link text</a>
 ```
 
-## Images
+## Insert an image
+
+To insert an image into your content, use the following:
 
-`!``[]``(LOCATION_OF_IMAGE)`
+```bash
+<%= image_tag "image location", { :alt => 'image alt text' } %>
+```
+
+You must include alt text because of accessibility requirements.
 
 ## Insert a warning
 
 Use the warning text component when you need to warn users about something important, such as legal consequences of an action, or lack of action, that they might take.
 
-This warning text format is consistent with the [GOV.UK Design System warning text](https://design-system.service.gov.uk/components/warning-text/).
+This warning text format is consistent with the [GOV.UK Design System warning text][design-system].
 
-Insert the following Ruby code into your content file:
+To insert a warning, use the following:
 
 ```ruby
-<%%= warning_text('INSERT WARNING TEXT HERE') %>
+<%%= warning_text('insert warning text here') %>
 ```
+
+## Use the design system
+
+Use [design system](https://design-system.service.gov.uk/) elements in your documentation to ensure your content complies with accessibility regulations. 
```

Comment:
```suggestion
Use [GOV.UK Design System](https://design-system.service.gov.uk/) elements in your documentation to ensure your content complies with accessibility regulations. 
```

Reply:
I'm not sure what Design System elements we mean here? I can't think of many (or any?) elements that you'd use inside flat documentation.